.npmignore

```diff
@@ -0,0 +1,7 @@
+/assets
+/blueprints/.jshintrc
+/dev
+/docs
+/tests/!(helpers)
+/tmp
+/*.*
```

Comment:
@stefanpenner did you see the image here? https://github.com/ember-cli/ember-cli/pull/5356#issuecomment-172391663